principal_travel/static/js/script.js

```diff
@@ -102,20 +103,20 @@ var renderTable = function(tripData){
   });
 };
 
-var renderMap = function(tripData){
-  var map = L.map('map')
-  .setView([33, -8], 2);
 
+var renderMap = function(tripData){
   L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
     id: 'noonkay.f59d9773',
     // id: 'noonkay.pfg0o793',
     accessToken: 'pk.eyJ1Ijoibm9vbmtheSIsImEiOiJjaWo4cHhpa2UwMDFidXhseDg3eGMwejBuIn0.tBWxkbD9BloELWmccA1UyQ',
-    scrollwheel: false,
-    noWrap: false,
+    noWrap: true,
```

Comment:
I changed `noWrap` to `false` because I widened the width of the map, and in order for the map to take up the full width, it had to repeat. If we change it back to true, then it won't repeat/wrap and there will be extra gray space around the map.